README.md

```diff
@@ -1,173 +1,172 @@
+---
+editor_options: 
+  markdown: 
+    wrap: 72
+---
+
 [![Slack](https://img.shields.io/badge/slack-chat-orange.svg)](https://join.slack.com/t/kuwala-community/shared_invite/zt-l5b2yjfp-2cdhReNACXDkLhuh0jFzZw)
 ![License](https://img.shields.io/github/license/kuwala-io/kuwala)
 
-<img src="./docs/images/kuwala_header_github.png" width="100%" />
+![](docs/images/header_github_kuwala_new.png)
 
 <p align="center">
-  <a href="https://join.slack.com/t/kuwala-community/shared_invite/zt-l5b2yjfp-2cdhReNACXDkLhuh0jFzZw"><img src="./docs/images/slack_join_button_github.png" height="96" /></a>
-  <a href="https://github.com/orgs/kuwala-io/projects/3/views/6"><img src="./docs/images/roadmap_button_github.png" height="96" /></a>
-</p>
 
-Kuwala is **the no-code data platform for BI analysts and engineers enabling you to build powerful analytics workflows.** 
-We are set out to bring **state-of-the-art data engineering tools** you love, such as 
-[Airbyte](https://github.com/airbytehq/airbyte), [dbt](https://github.com/dbt-labs/dbt-core), or 
-[Great Expectations](https://github.com/great-expectations/great_expectations) together in one intuitive interface built 
-with [React Flow](https://github.com/wbkd/react-flow).
+<a href="https://join.slack.com/t/kuwala-community/shared_invite/zt-l5b2yjfp-2cdhReNACXDkLhuh0jFzZw"><img src="./docs/images/slack_join_button_github.png" height="96"/></a>
+<a href="https://github.com/orgs/kuwala-io/projects/3/views/6"><img src="./docs/images/roadmap_button_github.png" height="96"/></a>
```

Comment:
Buttons are not centered anymore. I think you need to remove the empty lines above the <a> tags.

![image](https://user-images.githubusercontent.com/28758040/170075690-e5472931-d383-4c29-86b5-9d22fdd6a02e.png)